js/tests/.eslintrc.json

```diff
@@ -28,6 +28,7 @@
     "global-require": "off",
     "no-process-env": "off",
     "no-process-exit": "off",
+    "no-sync": "off",
```

Comment:
Isn't this off by default?

Reply:
unfortunately no 😄

Reply:
Weird, doessn't seem enabled here https://eslint.org/docs/rules/#nodejs-and-commonjs

Reply:
That's because of this line here : https://github.com/twbs/bootstrap/blob/v4-dev/js/.eslintrc.json#L122